src/MainWindow.vala

```diff
@@ -348,36 +325,14 @@ public class AppCenter.MainWindow : Hdy.ApplicationWindow {
             }
         });
 
-        delete_event.connect (() => {
+        close_request.connect (() => {
             ((AppCenter.App) application).request_background.begin (() => destroy ());
 
-            return Gdk.EVENT_STOP;
+            return false;
```

Comment:
Maybe this can be moved into the override bool close_request?
Also I think this should probably return true?

Reply:
Yeah I'm not sure why this was separate in master. Moved!